beaker-tests/README.md

```diff
@@ -0,0 +1,54 @@
+How to run tests
+================
+
+The following command will setup your system, setup a Fedora virtual machine and run tests from the `./tests/` subdirectory. You need to be root and the command will modify your system, i.e. install some packages, overwrite `/etc/rpkg/*.conf` configs and `/etc/rpkg.conf`, write an entry into `/etc/hosts`, and generates root ssh public key if not generated yet. It might be therefore a good idea to run this command inside another virtual machine (try with nested virtualization enabled) or a container:
+
+```
+# ./run.sh
+```
+
+The setup part for most of the host system modifications and creation of the virtual machine is done by `./setup.sh` script that is invoked from `run.sh` and is the part that requires root privileges. Use `cat setup.sh` to see what the script does. You can choose not to run the script during `./run.sh` execution if you have run it before (either through `./run.sh` or `./setup.sh` directly). To do this, pass `-x` switch to `./run.sh`:
+
+```
+# ./run.sh -x
+```
+
+To run the tests for CentOS-7 (with epel-7 repos enabled) instead of Fedora, you can use:
+
+```
+# DISTGIT_FLAVOR=dist-git-centos-7 ./run.sh
+```
+
+This will spawn a new CentOS-7/epel-7 virtual machine.
+
+To do the same for CentOS-8 (with epel-8 repos enabled), you can use:
+
+```
+# DISTGIT_FLAVOR=dist-git-centos-8 ./run.sh
+```
+
+Between each run, you need to remove `pkgs.example.org` record from `/root/.ssh/known_hosts` and you should also clean-up `/etc/hosts` from stale `pkgs.example.org` records.
```

Comment:
This is a bit cumbersome. Not saying that it should be done within this PR, but sometime we should do this automatically when the `run.sh` script finishes (or starts).

Reply:
That's true, the test-suite would deserve a little bit of care but I would do it separately as an improvement.